### PR TITLE
Thefunzone

### PR DESCRIPTION
**Pull request by renross**

### CODE REVIEW

File: src/commands/fun/thisisfine.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { Message } from 'discord.js';
+import { Command } from '@lib/types/Command';
+
+export default class extends Command {
+
+	description = 'everything is fine...probably';
```
Comment (joshualyon7) [inline src/commands/fun/thisisfine.ts:6]: Space after ...

File: src/commands/fun/thisisfine.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+	aliases = ['tif'];
+
+	run(msg: Message): Promise<Message> {
+		return msg.channel.send({ files: [{
+			attachment: `${__dirname}../../../../../assets/images/thisisfine.png`,
```
Comment (joshualyon7) [inline src/commands/fun/thisisfine.ts:11]: Could we possibly alias assets? Like we do for `@lib` and `@root`

So then we could do... 
```suggestion
			attachment: `@assets/images/thisisfine.png`,
```

(and the same for tif lines [29](https://github.com/ud-cis-discord/SageV2/pull/208/files#diff-15a77f2575ef549c61f7fa5d38eeab07596ff5a944dd0e1abb2060429e6535b9R29) and [34](https://github.com/ud-cis-discord/SageV2/pull/208/files#diff-15a77f2575ef549c61f7fa5d38eeab07596ff5a944dd0e1abb2060429e6535b9R34))

File: src/commands/fun/coinflip.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
```
Comment (joshualyon7) [inline src/commands/fun/coinflip.ts:1]: Why was this necessary?

File: src/commands/fun/coinflip.ts [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+const COIN_FLIP = ['You got: Heads!', 'You got: Tails!'];
+
+export default class extends Command {
+
+	description = 'Have Sage flip a coin for you!';
+	aliases = ['flip', 'coin', 'cf'];
+
+	async run(msg: Message) {
+		const coinFlip = await msg.channel.send('flipping...');
+		/*
```
Comment (joshualyon7) [inline src/commands/fun/coinflip.ts:16]: Probably delete this comment for cleanliness

**Review by joshualyon7 — CHANGES_REQUESTED** [review]

File: src/commands/fun/coinflip.ts (hunk 3 above)
Comment (renross) [reply src/commands/fun/coinflip.ts:1]: i didn't have a return type, fixed it

File: src/commands/fun/thisisfine.ts (hunk 2 above)
Comment (renross) [reply src/commands/fun/thisisfine.ts:11]: that one is Ben's fault

Comment (renross) [reply src/commands/fun/thisisfine.ts:11]: i fixed it in s;f too

Comment (BenSegal855) [reply src/commands/fun/thisisfine.ts:11]: Iirc the aliases won't work for file uploads because they result in a relative path and an absolute path is used. That's why `s;f` uses this same method.

Comment (renross) [reply src/commands/fun/thisisfine.ts:11]: yeah josh

Comment (joshualyon7) [reply src/commands/fun/thisisfine.ts:11]: Ouch, that's obnoxious

File: src/commands/fun/thisisfine.ts (hunk 1 above)
Comment (joshualyon7) [reply src/commands/fun/thisisfine.ts]: I guess I'll concede

File: src/commands/fun/thisisfine.ts (hunk 2 above)
Comment (BenSegal855) [reply src/commands/fun/thisisfine.ts]: I don't remember the whole problem but I know there was an issue with using the aliases in the Jenkins build because they were relative. I think it was something along the lines of having the aliases be relative to the root of the project but when used in Jenkins they were relative to the file or something like that. I just remember the fix was to use absolute paths and not worry about it.

Comment (joshualyon7) [reply src/commands/fun/thisisfine.ts]: Fair enough

**Review by joshualyon7 — CHANGES_REQUESTED** [review]
Code looks good, just merge into dev instead of main

**Review by joshualyon7 — APPROVED** [review]
